rpcq/_base.py

```diff
@@ -42,47 +43,15 @@ class UnknownMessageType(Exception):
     """Raised when trying to decode an unknown message type."""
 
 
-class Message(object):
+class Message:
     """
     Base class for messages.
     """
 
-    def __repr__(self):
-        return "{}({})".format(
-            self.__class__.__name__,
-            ", ".join("{}={}".format(k, repr_value(v))
-                      for k, v in sorted(self.asdict().items(), key=lambda kv: kv[0])))
-
-    def __getitem__(self, item):
-        return self.asdict()[item]
-
-    def items(self):
-        return self.asdict().items()
-
-    def get(self, key, default):
-        return self.asdict().get(key, default)
-
-    def __eq__(self, other):
-        return type(self) == type(other) and self.astuple() == other.astuple()
-
-    def __copy__(self):
-        return self.__class__(**self.asdict())
-
-    def __deepcopy__(self, memo):
-        ret = self.__class__(**deepcopy(self.asdict(), memo=memo))
-        memo[id(self)] = ret
-        return ret
-
-    def copy(self, **kwargs):
-        """
-        Create a copy with (optionally) substituted values.
-
-        :param kwargs: Any fields that should be substituted.
-        :return: A new message object of the same class.
-        """
-        values = self.asdict()
-        values.update(kwargs)
-        return self.__class__(**values)
+    def __post_init__(self):
```

Comment:
Why do we have to do this ourselves?

Reply:
That's a question for @stevenheidel , since I copy-pasted this file from his internal PR.

Reply:
I can't for the life of me figure out why that was. Do things break if we remove it?

Reply:
Nothing seems to break... I'm going to follow @stevenheidel's suggestion, and we can reinsert it if we discover what we wanted it for later on.